spec/cucumber/core/report/summary_spec.rb

```diff
@@ -61,14 +61,61 @@ module Report
             expect(@summary.test_cases.total).to eq(1)
           end
 
-          it 'handles flaky test cases' do
-            allow(test_case).to receive(:==).and_return(false, true)
-            event_bus.send(:test_case_finished, test_case, failed_result)
-            event_bus.send(:test_case_finished, test_case, passed_result)
+          context 'handles flaky test cases' do
+            context 'with following skip test cases' do
+              it do
```

Comment:
as an FYI - we're trying to decouple / declutter specs with multiple assertions. I appreciate the ones above have multiple assertions but if you can get things close to single assertion that would help me immensely.

Also the tests don't appear to have any names inside the it statement

Reply:
@luke-hill Updated, thanks!